lib/tasks/represent_downstream.rake

```diff
@@ -15,13 +15,14 @@ namespace :represent_downstream do
   end
 
   desc "
-  Represent downstream for a specific document_type
+  Represent downstream for individual or multiple document_types
   Usage
-  rake 'represent_downstream:document_type[:document_type]'
+  rake 'represent_downstream:document_type[:document_types]'
   "
-  task :document_type, [:document_type] => :environment do |_t, args|
+  task :document_types, [:document_types] => :environment do |_t, args|
```

Comment:
I think we should keep the task name as `document_type` as that might break some other users of this task.

Reply:
great point